src/inmanta/protocol/methods_v2.py

```diff
@@ -56,16 +56,18 @@ def project_delete(id: uuid.UUID) -> None:
 
 
 @typedmethod(path="/project", operation="GET", client_types=[ClientType.api], api_version=2)
-def project_list() -> List[model.Project]:
+def project_list(details: bool = False) -> List[model.Project]:
```

Comment:
I am wondering whether we should call the argument`environment_details` on the project endpoints. Otherwise we get naming problems if we ever want to have a `details` setting for the project as well. But I am not sure about this. Wdyt?